hoot-core/src/main/cpp/hoot/core/io/ApiDb.cpp

```diff
@@ -1005,11 +1004,20 @@ boost::shared_ptr<QSqlQuery> ApiDb::selectElements(const ElementType& elementTyp
   {
     _selectQueries[elementTableName].reset(new QSqlQuery(_db));
     _selectQueries[elementTableName]->setForwardOnly(true);
```

Comment:
Found an unexpected behavior by the ApiDbReader in that it isn't always returning elements sorted by ID, like I thought it was.  Doesn't matter, though, since we can sort results after the fact....just wanted to make note of it.